frontend/src/layout/header/Search.tsx

```diff
@@ -33,35 +34,28 @@ export const SearchField: React.FC<SearchFieldProps> = ({ variant }) => {
     // initial-loading phase is done.
     const disabled = useUser() === "unknown";
 
-    // Register global shortcut to focus search bar
-    useEffect(() => {
-        const handleShortcut = (ev: KeyboardEvent) => {
-            // Clear input field. See comment inside `handleNavigation` function
-            // in `routes/Search.tsx` for explanation. I'd rather do this here than
-            // needing to pass the input ref.
-            if (ev.key === "Escape" && ref.current) {
-                ref.current.value = "";
-            }
-            // If any element is focussed that could receive character input, we
-            // don't do anything.
-            if (/^input|textarea|select|button$/i.test(document.activeElement?.tagName ?? "")) {
-                return;
-            }
-
-            // With ctrl and meta key, this could be bound to some other
-            // shortcut (ctrl+s) that we want to ignore.
-            if (ev.ctrlKey || ev.metaKey) {
-                return;
-            }
+    useShortcut(
+        SHORTCUTS.general.search.keys,
+        () => ref.current?.focus(),
+        {
+            preventDefault: true,
+            // Will cause infinite rerenders if not memoized, hence the `useCallback`.
+            ignoreEventWhen: useCallback(
+                () => /^input|textarea|select|button$/i.test(document.activeElement?.tagName ?? ""),
+                [],
+            ),
+        },
+    );
 
-            if (ev.key === "s" || ev.key === "S" || ev.key === "/") {
-                ref.current?.focus();
+    useShortcut(
+        "escape",
+        () => {
+            if (ref.current) {
+                ref.current.value = "";
             }
```

Comment:
That now always clears on escape, even if the search field isn't focused. Not a huge deal, but maybe weird? Do we even need this shortcut anymore? Might be a remnant of instant search time? Ye maybe just remove the escape thing here.